pkg/cloud/services/ec2/helper_test.go

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Do we still need this file?

Reply:
Yes @Ankitasw this file will contains all the common test helper functions like `setupScheme()`, `newAWScluster()`, etc that could be use across all files within `ec2` package.